go.mod

```
@@ -437,7 +437,8 @@ replace github.com/lestrrat-go/jwx/v2 => github.com/lestrrat-go/jwx/v2 v2.0.9-0.
 // Uncomment for local development for testing with changes in the components-contrib repository.
 // Don't commit with this uncommented!
 //
-// replace github.com/dapr/components-contrib => ../components-contrib
+//replace github.com/dapr/components-contrib => ../components-contrib
+
```

Tests will fail unless this is unchanged

```suggestion
// replace github.com/dapr/components-contrib => ../components-contrib
```